diff -urNp -x '*.orig' guile-cairo-1.10.0.org/guile-cairo/guile-cairo.h guile-cairo-1.10.0/guile-cairo/guile-cairo.h
--- guile-cairo-1.10.0.org/guile-cairo/guile-cairo.h	2017-03-20 11:00:26.000000000 +0100
+++ guile-cairo-1.10.0/guile-cairo/guile-cairo.h	2022-03-12 14:09:32.041432334 +0100
@@ -24,6 +24,11 @@
 #include <cairo.h>
 #include <libguile.h>
 
+#include <cairo.h>
+#if CAIRO_HAS_SVG_SURFACE
+#include <cairo-svg.h>
+#endif
+
 #include <guile-cairo-smob-types.h>
 #include <guile-cairo-enum-types.h>
 #include <guile-cairo-vector-types.h>
